test/extended/authentication.sh

```diff
@@ -59,34 +59,135 @@ oc new-project openldap
 # create all the resources we need
 oc create -f test/extended/fixtures/ldap
 
+is_event_template=(                      \
```

Comment:
I used tabs to align the left side and spaces to align the `\` ... unclear why this is not working.